src/snowflake/cli/plugins/containers/services/manager.py

```diff
@@ -23,8 +30,29 @@ def create(
             WITH
             MIN_INSTANCES = {num_instances}
             MAX_INSTANCES = {num_instances}
-            """
-        )
+            AUTO_RESUME = {auto_resume}
+            """.splitlines()
+
+        if external_access_integrations:
+            external_access_integration_list = ",".join(
+                f"{e}" for e in external_access_integrations
+            )
+            query.append(
+                f"EXTERNAL_ACCESS_INTEGRATIONS = ({external_access_integration_list})"
+            )
+
+        if query_warehouse:
+            query.append(f"QUERY_WAREHOUSE = {query_warehouse}")
+
+        if tags:
+            tag_list = ",".join(f"{t.name}={t.value_string_literal()}" for t in tags)
+            query.append(f"TAG ({tag_list})")
+
+        if comment:
+            query.append(f"COMMENT = {comment}")
+
+        query_string = "\n".join([q.strip() for q in query if q.strip()])
```

Comment:
`[]` can be omitted here.
```suggestion
        query_string = "\n".join(q.strip() for q in query if q.strip())
```

Also this is the same code as above with some logic. It could be a candidate for some local helper function.